network-api/networkapi/wagtailpages/translation.py

```diff
@@ -1,3 +1,5 @@
+# TODO: REmove this and other translation.py files.
+# They aren't needed but keep coming back when we merge master into our localization branch
```

Comment:
Added this because it keeps coming back. It's not doing any harm being here, so I'm leaving this for "final cleanup" when localization gets merged into the master branch.